apps/cookbook/src/app/shared/api-description/api-description-properties/api-description-properties.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+
+export interface ApiDescriptionProperty {
+  name: string;
+  description?: string;
+  defaultValue?: string;
+  type?: string[];
+  preserveTypeWhitespaces?: boolean;
+}
+
+export interface ApiDescriptionPropertyColumns {
+  Name: string;
+  Description?: string;
+  Type?: string;
+  Default?: string;
```

Comment:
We use camel-casing:
```suggestion
  name: string;
  description?: string;
  type?: string;
  default?: string;
```